src/react/account/__tests__/UpdateAccountPolicy.test.tsx

```diff
@@ -55,27 +94,34 @@ describe('UpdateAccountPolicy', () => {
     expect(cancelButton).toBeInTheDocument();
   });
   it('should display fields policy name and ARN not empty', async () => {
-    render(<UpdateAccountPolicy/>, {
+    reduxRender(<UpdateAccountPolicy />, {
       wrapper,
     });
+    await waitForElementToBeRemoved(
+      () => [...screen.queryAllByText(/Loading/i)],
+      { timeout: 8000 },
+    );
 
     const policyNameLabel = screen.getByTestId('policyNameLabel');
     expect(policyNameLabel).not.toBeEmptyDOMElement();
 
     const policyARNLabel = screen.getByTestId('policyARNLabel');
     expect(policyARNLabel).not.toBeEmptyDOMElement();
```

Comment:
I think the test could be improved a bit here.
We could check if the expected `name`, `arn` and `policy document` are rendered correctly.